io/io/src/TFile.cxx

```diff
@@ -4031,6 +4033,18 @@ TFile *TFile::Open(const char *url, Option_t *options, const char *ftitle,
    }
 
    TString expandedUrl(url);
+
+#ifndef WIN32
+   { // if url is a regular file with XRootD URL extended attribute, redirect
+      struct stat st;
+      if (stat(url, &st) == 0 && !S_ISDIR(st.st_mode)) {
+         char xurl[PATH_MAX];
+         if (getxattr(url, "eos.url.xroot", xurl, PATH_MAX) > 0)
+            expandedUrl = TString(xurl);
```

Comment:
Does this work?
```suggestion
            expandedUrl = xurl;
```

Reply:
Remnant from when I had this change in `TUrl`, will fix in next push.